modules/repository/env.go

```diff
@@ -25,11 +25,16 @@ const (
 	EnvKeyID        = "GITEA_KEY_ID" // public key ID
 	EnvDeployKeyID  = "GITEA_DEPLOY_KEY_ID"
 	EnvPRID         = "GITEA_PR_ID"
+	EnvPRAction     = "GITEA_PR_ACTION"
```

Comment:
I think it should be called `GITEA_PUSH_TRIGGER`

* GITEA_PUSH_TRIGGER= (empty, default)
* GITEA_PUSH_TRIGGER=pr-update-with-base
* GITEA_PUSH_TRIGGER=pr-merge-to-base

etc

Reply:
3e320f865f1c93e1ce646da74f5cacc6c404e3e1